File: LeetCode/LeetCode 674. 最长连续递增序列.cpp
class Solution {
public:
    int findLengthOfLCIS(vector<int>& nums) {
        if(!nums.size()) return 0;
        int ans = 1, now = 1;
        for(int i = 1; i < nums.size(); i++) {
            if(nums[i] > nums[i - 1]) now++;
            else {
                ans = max(ans, now);
                now = 1;
            }
        }
        ans = max(ans, now);
        return ans;
    }
};

File: LeetCode/LeetCode 674. 最长连续递增序列.py
class Solution:
    def findLengthOfLCIS(self, nums: List[int]) -> int:
        if len(nums) == 0:
            return 0
        ans, now = 1, 1

        for i in range(1, len(nums)):
            if nums[i] > nums[i - 1]:
                now += 1
            else:
                ans = max(ans, now)
                now = 1

        ans = max(ans, now)
        return ans